### PR TITLE
Added mongodb connection, work in progress on entities

### PR DESCRIPTION
**Pull request by okasi**

### CODE REVIEW

**Comment by okasi** [comment]
added husky w/ pre-commit format command as well

File: src/entity/User.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { Column, Entity, PrimaryGeneratedColumn, OneToMany } from 'typeorm'
+import { Room } from './Room'
+
+@Entity()
+export class User {
+  @PrimaryGeneratedColumn()
```
Comment (aminmus) [inline src/entity/User.ts:6]: If we are using mongodb it seems we are gonna need to change @PrimaryGeneratedColumn(), see https://github.com/typeorm/typeorm/blob/master/docs/mongodb.md

File: src/index.ts [hunk 2]
```diff
@@ -1,23 +1,34 @@
...
-  dotenv.config();
+  await createConnection({
+    type: 'mongodb',
+    url: `${process.env.MONGO_DB_URL}`,
+  })
 
-  const schema = await buildSchema({
-    resolvers: [],
-  });
+  // const schema = await buildSchema({
```
Comment (aminmus) [inline src/index.ts:17]: Can remove these comments possibly

**Comment by aminmus** [comment]
We probably have to discuss how bots (like the covid-19 bot) should fit in the application architecture and how their data should be stored.

**Pull request merged**